frontend/public/components/_secret.scss

```diff
@@ -0,0 +1,31 @@
+.co-create-secret-form {
```

Comment:
BEM discourages nesting, and we should give the classes appropriate names for BEM.

http://getbem.com/

Reply:
Style properties should be alphabetized.

Reply:
Saw nesting in our code-base so thought it's ok, but will try to avoid it ! Thanks :)